cli/src/lib.rs

```diff
@@ -708,11 +631,15 @@ mod tests {
                 dir.path().join("storage")
             );
             assert_eq!(
-                config.snapshot.dir_path.absolutize()?,
+                config.snapshot.store_path.absolutize()?,
                 dir.path().join("snapshots")
             );
             assert_eq!(
-                config.telemetry.file.expect("Should be set").absolutize()?,
+                config
+                    .dev_telemetry
```

Comment:
```suggestion
                config
                    .telemetry
```

this used to be `telemetry`. Is this change ok?

Reply:
It used to be `telemetry.file`. `file` is a part of `telemetry.dev` namespace now, which is parsed into `.dev_telemetry` structure now. Therefore, it's not a mistake.